api/src/main/java/io/opencensus/common/ServerStats.java

```diff
@@ -19,7 +19,11 @@
 import com.google.auto.value.AutoValue;
 import javax.annotation.concurrent.Immutable;
 
-/** A representation of stats measured on the server side. */
+/**
+ * A representation of stats measured on the server side.
+ *
+ * @since 0.15
```

Comment:
s/0.15/0.16, same for all occurrence, since 0.15 is already released.

Reply:
done.